DSTV.Net.Test/DSTV.Net.Test.csproj

```diff
@@ -3,6 +3,7 @@
     <PropertyGroup>
         <IsPackable>false</IsPackable>
         <Title>Testclass for DSTV.Net</Title>
+		<TargetFrameworks>net6.0;net7.0;</TargetFrameworks>
```

Comment:
Does this also need a `netstandard2.0`?

Reply:
This is explained in the PR description. Added netstandard here breaks unit tests for non Windows operating systems as the xunit runners will be restored using .NET Framework 4.7.3.